src/main/java/com/github/onsdigital/babbage/metrics/Metrics.java

```diff
@@ -0,0 +1,12 @@
+package com.github.onsdigital.babbage.metrics;
+
+public interface Metrics {
+
+    void incPublishDatePresent();
+    void incPublishDateNotPresent();
+    void incPublishDateInFuture();
+    void incPublishDateTooFarInPast();
+    void incPublishDateTooFarInFuture();
+    void setCacheExpiryTime(Double expiryTime);
```

Comment:
needed ?

Reply:
One of the tasks on the ticket says "Ensure that when a cache header is set in Babbage then this is correctly graphed on the dashboard". If that task is still valid then this is needed (for recording the expiry time). But I'm happy to delete that task?

Reply:
removed now